openedx/core/djangoapps/oauth_dispatch/dot_overrides/views.py

```diff
@@ -86,7 +87,7 @@ def get(self, request, *args, **kwargs):
                     uri, headers, body, status = self.create_authorization_response(
                         request=self.request, scopes=" ".join(scopes),
                         credentials=credentials, allow=True)
-                    return HttpResponseUriRedirect(uri)
+                    return HttpResponseUriRedirect(uri, application.get_allowed_schemes())
```

Comment:
Why was this change needed?

Reply:
@douglashall ?

Reply:
The `HttpResponseUriRedirect.__init__` signature changed with the new version of DOT.